ext/kv/Cargo.toml

```diff
@@ -32,6 +32,7 @@ serde_json.workspace = true
 tokio.workspace = true
 url.workspace = true
 uuid = { workspace = true, features = ["serde"] }
+termcolor = "1.1.3"
```

Comment:
Maybe hoist it to top-level `Cargo.toml` and use `termcolor.workspace = true`. That way we won't end up with two versions by accident

Reply:
Will do